tests/flash_attn_wrapper.py

```diff
@@ -0,0 +1,89 @@
+from typing import Optional, List, Tuple
```

Comment:
Is this file necessary now that you've added torch bindings?

Reply:
I added it more because I wanted to keep the test intact so it can be kept in sync with the vllm file. But yeah the wrapper is not necessary